core/tx_pool.go

```diff
@@ -227,6 +228,7 @@ type TxPool struct {
 	all     *txLookup                    // All transactions to allow lookups
 	priced  *txPricedList                // All transactions sorted by price
 
+	newTxsCh  chan []*types.Transaction
```

Comment:
fixme lint problem like as follow
~~~diff
--- core/tx_pool.go     2019-01-29 12:06:03.734814044 +0900
+++ txx 2019-01-29 12:06:19.046511165 +0900
@@ -228,8 +228,8 @@
        all     *txLookup                    // All transactions to allow lookups
        priced  *txPricedList                // All transactions sorted by price

-       newTxsCh  chan []*types.Transaction
-       wg sync.WaitGroup // for shutdown sync
+       newTxsCh chan []*types.Transaction
+       wg       sync.WaitGroup // for shutdown sync

        homestead bool
 }
~~~